RCNN_tf.py

```diff
@@ -10,9 +10,14 @@
 __author__ = "Adrian Chmielewski-Anders, Leo Zeitler & Bas Straathof"
 
 import sys
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2' # suppress warnings
+
 import tensorflow as tf
+tf.logging.set_verbosity(tf.logging.ERROR) # suppress warnings
```

Comment:
Are both lines
`os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'``and
`tf.logging.set_verbosity(tf.logging.ERROR) `
really necessary to suppress the warnings or is it more duplicating

Reply:
They are both necessary, since one suppresses warnings and the other one suppress the "I, Tensorflow did .... bla bla bla" output.